Library/Formula/geos.rb

```diff
@@ -13,6 +13,12 @@ class Geos < Formula
 
   option :universal
   option :cxx11
+  option 'with-php', 'Build the PHP extension'
+  option 'with-python', 'Build the Python extension'
+  option 'with-ruby', 'Build the ruby extension'
+  option 'use-homebrew-php', 'Use Homebrew PHP' if build.include?('with-php')
```

Comment:
Style notes:
* we follow the [Github Ruby style guide](https://github.com/styleguide/ruby) for new commits, so please use double quotes everywhere.
* use `build.with?("php")` instead of `build.include?("with-pip")`